backend/src/models/player.model.ts

```diff
@@ -3,11 +3,13 @@ import { Document, Schema } from "mongoose";
 export interface Player extends Document {
   nickname: string;
   punchlines?: string[];
-  score?: number;
+  score: number;
+  new: boolean;
 }
 
 export const PlayerSchema: Schema = new Schema({
   nickname: { type: String, required: true },
   punchlines: [String],
-  score: Number,
+  score: { type: Number, default: 0 },
+  new: { type: Boolean, default: true },
```

Comment:
This change does look necessary 👍

Reply:
Yep needed scores to default to zero & track new players that have not connected to Socket.IO yet